1 Table.sql

```diff
@@ -0,0 +1,49 @@
+use ConfectioneryDB
+go 
+
+drop table if exists Confectionery
+go 
+
+create table dbo.Confectionery(
```

Comment:
A clearer name for such a table would be Orders.

Reply:
I changed the table name to orders